flink-runtime/src/main/java/org/apache/flink/runtime/operators/resettable/AbstractBlockResettableIterator.java

```diff
@@ -183,4 +183,24 @@ protected T getNextRecord() throws IOException {
 		}
 	}
 
+	public void resetForIterativeTasks() {
```

Comment:
This class already has a method called `reset`. Why do we need to add another resetting method?